harness/web/src/styles.css

```
@@ -684,19 +691,212 @@ code,
+  content: "— ";
+  color: var(--ink-3);
+  font-style: normal;
+}
+
+.md code {
+  font-family: var(--mono);
+  font-size: 0.88em;
+  background: var(--paper-3);
+  padding: 1px 5px;
+  color: var(--ink);
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)